tensorflow/contrib/seq2seq/python/ops/helper.py

```diff
@@ -443,7 +443,7 @@ def sample(self, time, outputs, state, name=None):
     # Outputs are logits, use argmax to get the most probable id
     if not isinstance(outputs, ops.Tensor):
       raise TypeError("Expected outputs to be a single Tensor, got: %s" %
```

Comment:
OK. In that case never mind. On second thought, the alternative I suggested was not 100% clear, either. What you have is fine and shorter.